app/views/shared/_header.html.haml

```diff
@@ -20,3 +26,5 @@
               %li.divider-vertical
               = render "shared/message_summary"
               %li= link_to t('pages.logout'), sign_out_user_path
+          .hidden-with-nav
+            = render :partial => '/shared/nav/nav_links'
```

Comment:
We would lean towards `= render "shared/nav/nav_links`. So not using `partial` and using double quotes.